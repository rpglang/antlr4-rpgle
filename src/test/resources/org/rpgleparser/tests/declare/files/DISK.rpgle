       DCL-F NAMSTR DISK(*EXT) USAGE(*INPUT) KEYED;
