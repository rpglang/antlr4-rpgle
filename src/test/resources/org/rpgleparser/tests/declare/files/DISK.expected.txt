/*===TOKENS===*/
FS_FreeFile              <DCL-F>
ID                       <NAMSTR>
KEYWORD_DISK             <DISK>
'('                      <(>
SPLAT_EXT                <*EXT>
')'                      <)>
KEYWORD_USAGE            <USAGE>
'('                      <(>
SPLAT_INPUT              <*INPUT>
')'                      <)>
KEYWORD_KEYED            <KEYED>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (fspec
      DCL-F
      (filename NAMSTR)
      (fs_keyword (keyword_disk DISK ( *EXT )))
      (fs_keyword (keyword_usage USAGE ( (symbolicConstants *INPUT) )))
      (fs_keyword (keyword_keyed KEYED))
      ;
    )
  )
)
/*======*/